core/shared/src/main/scala/zio/internal/FiberContext.scala

```diff
@@ -1067,6 +1067,20 @@ private[zio] final class FiberContext[E, A](
     // For improved fairness, we resume in order of submission:
     observers.reverse.foreach(k => k(result))
   }
+
+  override def toString: String =
+    s"""
+       |FiberContext(identityHashCode=${System.identityHashCode(this)})
+       |  fiberId:         $fiberId
+       |  platform:        
```

Comment:
Shouldn't you delegate to the underlying `toString` here? This version is going to be more readable and I haven't looked at the existing pretty-printer, so feel free to ignore.

Reply:
Updated